frame/dvm/ethereum/src/lib.rs

```diff
@@ -302,13 +302,19 @@ pub mod pallet {
 	}
 
 	#[pallet::event]
-	#[pallet::generate_deposit(pub(super) fn deposit_event)]
+	#[pallet::generate_deposit(pub fn deposit_event)]
 	/// Ethereum pallet events.
-	pub enum Event {
+	pub enum Event<T: Config> {
 		/// An ethereum transaction was successfully executed. \[from, to/contract_address, transaction_hash, exit_reason\]
 		Executed(H160, H160, H256, ExitReason),
 		/// DVM Transfer succeeded. \[from, to, value\]
 		DVMTransfer(H160, H160, U256),
+		/// Withdraw RING from DVM to darwinia account. \[sender, destination, value\]
+		RingBack(H160, T::AccountId, U256),
```

Comment:
Do we need to rename the `T::AccountId` to `AccountId`

Reply:
What's the difference after renaming?

Reply:
Looks like we don't need to rename the type after metadata v14.